packages/admin/dashboard/src/components/data-table/hooks/general/use-data-table-date-filters.tsx

```diff
@@ -0,0 +1,101 @@
+import { createDataTableFilterHelper } from "@medusajs/ui"
+import { useMemo } from "react"
+import { useTranslation } from "react-i18next"
+import { useDate } from "../../../../hooks/use-date"
+
+const filterHelper = createDataTableFilterHelper<any>()
+
+const useDateFilterOptions = () => {
+  const { t } = useTranslation()
+
+  const today = useMemo(() => {
+    const date = new Date()
+    date.setHours(0, 0, 0, 0)
+    return date
+  }, [])
+
+  return useMemo(() => {
+    return [
+      {
+        label: t("filters.date.today"),
+        value: {
+          $gte: today.toISOString(),
+        },
+      },
+      {
+        label: t("filters.date.lastSevenDays"),
+        value: {
+          $gte: new Date(
+            today.getTime() - 7 * 24 * 60 * 60 * 1000
```

Comment:
sugg: not necessarily for change, but we do have date-fns in the deps already to do something like - `subDays(date, 7)`

Reply:
True, good point!